environments/dev-env/README.md

```diff
@@ -0,0 +1,395 @@
+## Required Packages:
+   - ko: https://ko.build/install/ 
+   - gcc
+   - docker 
+   - jq
+   - make
+   - go (version expected 1.19)
+   - kind
+   - kubectl  
+
+
+## Supported OS Platforms 
+  - Linux
+  - MacOS
+  - Windows WSL/Ubuntu
+
+
+Run the following script to install the required package (Linux or MacOS ):
+
+```bash
+./install_req.sh
+```
+
+Alternatively, run the following command for MacOS platform:
+
+```bash
+brew install ko gcc jq make go@1.19 kind kubectl
+```
+
+For Windows WSL/Ubuntu platform, follow the instructions [here](docs/README.md)
+
+
+## Quickstart
+
+#### 1. Clone this repo:
+
+```bash
+git clone https://github.com/kcp-dev/edge-mc
+```
+
+#### 2. Change into the following directory path:
+
+```bash
+cd edge-mc/environments/dev-env
+```
+
+#### 3. Experiment with the kcp-edge 2023q1 PoC example scenarios:
```

Comment:
Is there actually any experimentation here, or is it just watching stuff scroll past on the console?